src/core/server/elasticsearch/legacy/scoped_cluster_client.ts

```diff
@@ -69,7 +69,7 @@ export class LegacyScopedClusterClient implements ILegacyScopedClusterClient {
     if (this.auditor) {
       this.auditor.add({
         message: endpoint,
-        type: 'elasticsearch.call.internalUser',
+        type: 'legacy.elasticsearch.call.internalUser',
```

Comment:
I wonder if we should actually use the same type as the new client. From a user's perspective, I would consider it a bug if they filtered their audit logs on `type: elasticsearch.call.*` and they didn't get these events in their results. I know there are slight differences in the data we can expose depending on the client (https://github.com/elastic/kibana/pull/71228/files#r461405740), but it seems we should try to keep the events as similar as possible.

@jportner thoughts?

Reply:
I tend to agree, but I think @thomheymann weigh in too.